platform/serviceConfig.yaml

```diff
@@ -0,0 +1,35 @@
+app: mlflow
+team: datascience
+
+
+webApps:
+  - name: mlflow
+    iamRole: cortex-basic
+    accessPolicy: external                       # external or internal
```

Comment:
What would `internal` do?

Reply:
Internal would not allow an internet facing load balancer, aka it would only be accessible from hosts within the production VPC. This is not what we want since we wouldn't be able to access it from VPN.